ratelimit/tests.py

```diff
@@ -389,6 +399,37 @@ def do_increment(request):
         assert do_increment(req), 'Request should be rate limited.'
         assert not_increment(req), 'Request should be rate limited.'
 
+    @override_settings(RATELIMIT_USE_CACHE='connection-errors')
+    def test_is_ratelimited_cache_connection_error(self):
```

Comment:
Mind dropping most of this test case and just calling `is_ratelimited` directly? The test it's copied from isn't a great test, and we can cover this case with something much simpler, more like the other test you added.